src/main/java/com/scayle/storefrontapi/model/Product.java

```diff
@@ -91,30 +85,12 @@ public class Product implements ApiObjectInterface {
     @SerializedName("firstLiveAt")
     String firstLiveAt;
 
-    /**
-    * 
-    */
-    @SerializedName("pricePromotionInfo")
-    Object pricePromotionInfo;
-
     /**
     * 
     */
     @SerializedName("priceRange")
     PriceRange priceRange;
 
-    /**
```

Comment:
Why were these fields removed here?

Reply:
These fields are not present in the JSON body that we receive from Scayle. When Gson tries to convert this JSON to a data model, exceptions occur.

Reply:
These fields are normally optional and will only be returned if they are requested via the `with` parameter.

Reply:
And you don't need these fields anywhere in your codebase?

Reply:
I don't need it, but the problem lies elsewhere. When using this data model and converting JSON with Gson, it throws an exception. You can verify this. The field is not present in the JSON response from Scayle.

Reply:
It does have the field when you provide `with=reductionRange`

Reply:
So these fields are valid and this is why I'm hesitant to remove them here

Reply:
Currently, we are using this dependency in our product and have practically discovered that this library needs to be corrected. I'm certain that these corrections are necessary. I strongly recommend checking this data model in connection with Scayle.

Reply:
https://dmg-live.storefront.api.scayle.cloud/v1/products?shopId=10001&with=reductionRange

This field does exist and as far as I know gson supports optional fields in the JSON response